tests/bootstrap.php

```diff
@@ -7,4 +7,4 @@
  * @license   http://opensource.org/licenses/BSD-2-Clause Simplified BSD License
  */
 
-require_once __DIR__ . '/../autoload_register.php';
+require dirname(__DIR__).'/vendor/autoload.php';
```

Comment:
There's no valid reason for this change.

Reply:
I may not agree. it offers greater ease of use and no need to regenerate the autoloader when you add classes, it's only a piece of advice.

Reply:
Oh nevermind, I missed the last part of the line. I'd still not use `dirname()` though in favor of `__DIR__ . '/../vendor/autoload.php'`… Thinking aobut this, the bootstrap file can actually go away completely, just define the vendor/autoload.php als bootstrap in the phpunit.xml.

Reply:
Yes, i usually like to define the vendor/autoload.php als bootstrap in the phpunit.xml.